tools/meson-vcs-tag.sh

```diff
@@ -4,14 +4,15 @@
 set -u
 set -o pipefail
 
-dir="${1:?}"
-fallback="${2:?}"
-version_tag="${3:-}"
+dir="${1:-.}"
+version_tag="${2:-}"
 
 if [ -n "${version_tag}" ]; then
     # If -Dversion_tag= was used, just use that without further changes.
     echo "${version_tag}"
 else
+    read -r -d '\n' project_version <"${dir}/meson.version"
```

Comment:
```suggestion
    read -r project_version <"${dir}/meson.version"
```

`read` reads the whole line by default